app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesPresenter.kt

```diff
@@ -25,4 +32,10 @@ class AllNotesPresenter(private val view: AllNotesView,
         adapter.cleanup()
     }
 
-}
\ No newline at end of file
+    private fun makeNoteDetailBundle(id: String?) : Bundle {
+        val args: Bundle = bundleFactory.createBundle()
+        screenBundleHelper.setNoteId(args, id)
+        screenBundleHelper.setTitle(args, R.string.note_detail_screen_title)
```

Comment:
Hmm.  This feels like it couples the screens a bit.  I wonder if there's a good clean way to decouple this.